### PR TITLE
support bigint64/biguint64 in TypedArray::ElementSize()

### PR DESCRIPTION
**Pull request by fs-eire**

### CODE REVIEW

File: napi-inl.h [hunk 1]
```diff
@@ -1628,6 +1628,10 @@ inline uint8_t TypedArray::ElementSize() const {
     case napi_float32_array:
       return 4;
     case napi_float64_array:
+#ifdef NAPI_EXPERIMENTAL
```
Comment (legendecas) [inline napi-inl.h:1631]: N-API BigInt support has been out of experimental and can be guarded with
```
#if (NAPI_VERSION >= 6)
```

Check out https://github.com/nodejs/node-addon-api/pull/697 for more contexts.

Comment (mhdawson) [inline napi-inl.h:1631]:
```suggestion
#if (NAPI_VERSION > 5)
```

File: napi-inl.h [hunk 2]
```diff
@@ -1628,6 +1628,10 @@ inline uint8_t TypedArray::ElementSize() const {
...
+    case napi_bigint64_array:
+    case napi_biguint64_array:
+#endif  // NAPI_EXPERIMENTAL
```
Comment (mhdawson) [inline napi-inl.h:1634]:
```suggestion
#endif  // NAPI_VERSION > 5
```

File: napi-inl.h [hunk 3]
```diff
@@ -1628,6 +1628,10 @@ inline uint8_t TypedArray::ElementSize() const {
     case napi_float32_array:
       return 4;
     case napi_float64_array:
+#if (NAPI_VERSION > 5)
+    case napi_bigint64_array:
+    case napi_biguint64_array:
+#endif
```
Comment (gabrielschulhof) [inline napi-inl.h:1634]: nit:
```suggestion
#endif  // (NAPI_VERSION > 5)
```

**Comment by mhdawson** [comment]
Landed as 4de23c9d6b66681390ba7d09e395390d159d308a

**Pull request closed without merging**